### ISSUE TITLE
New major version of protobuf.js

### ISSUE THREAD

**Issue by Crazy-Duck** [Arcana/node-dota2#402]
Check if we need to make changes if we want to migrate to the latest protobuf.js version

**Comment by Crazy-Duck** [comment]
This looks just plain awful '-_-
```
Dota2.schema.lookup("EGCBaseClientMsg").values.k_EMsgGCClientConnectionStatus
```
Anybody got an idea on how to make this prettier? Does anyone have tinkered with the new protobufjs yet? @howardchung , @jimmydorry ?

**Comment by jimmydorry** [comment]
Nope. Isn't it fine though? You can see how it was derrived, and how to extend it.

**Comment by Crazy-Duck** [comment]
I don't know, it just feels soo clunky :/ I mean, if you compare this
```
Dota2.schema.lookup("EGCBaseClientMsg").values.k_EMsgGCClientConnectionStatus
```
with this
```
Dota2.schema.EGCBaseClientMsg.k_EMsgGCClientConnectionStatus
```
I feel like the second one is more natural...

**Comment by Crazy-Duck** [comment]
Fixed in #409
